include/wx/qt/private/treeitemfactory.h

```diff
@@ -0,0 +1,123 @@
+/////////////////////////////////////////////////////////////////////////////
+// Name:        wx/treeitemfactory.h
+// Purpose:     Factory to create text edit controls for the tree items
+// Author:      Graham Dawes
+// Modified by:
```

Comment:
Very minor, but we don't use this header in the new files, it's a left over from the bad old pre-VCS days.

Reply:
Is the preferred header just the same but without the "Modified by" line, or is there something else we should use?

Reply:
Sorry for being unclear, the header is fine, it's just the "Modified by:" line that should be omitted.

Reply:
Cool. That has been fixed, along with the (one true) date and path issues below.